lib/src/ast/element.dart

```diff
@@ -97,6 +112,15 @@ abstract class ElementAst implements StandaloneTemplateAst {
   /// Reference assignments.
   List<ReferenceAst> get references;
 
+  /// Bananas assignments.
+  List<BananaAst> get bananas;
+
+  /// Star assignments.
+  List<StarAst> get stars;
+
+  /// Whitespaces
+  List<WhitespaceAst> get whitespaces;
```

Comment:
Yeah I agree. This is more of a temporary solution I wanted to implement as a placeholder before doing some more modifications on the AST level. Once I establish the AST to have better offset ranges, I'll remove the whitespaces list.